src/main/java/internet/store/service/impl/ProductServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package internet.store.service.impl;
+
+import internet.store.dao.ProductDao;
+import internet.store.lib.Inject;
+import internet.store.lib.Service;
+import internet.store.model.Product;
+import internet.store.service.Factory;
+import internet.store.service.ProductService;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    @Inject
+    private ProductDao productDao;
+
+    @Override
+    public Product create(Product product) {
+        return Factory.getProductDao().create(product);
```

Comment:
1) You don't need Factory with the new Injector;
2) You should use productDao variable since you are injecting it